### DISCUSSION TITLE
Can we change Column names?

### DISCUSSION THREAD

**Discussion by ashishagrawal121** [myliang/x-spreadsheet#628] [General]
Hi,

Can we change column names of sheet? Like in place of A,B,C... I want to replace it with some customized names.

--
Thanks,

**Comment by reviewher** [comment]
No, the labels are hardcoded https://github.com/myliang/x-spreadsheet/blob/master/src/component/table.js#L236

**Reply by nirajpandey3453** [reply]
can we enable /disable it
